src/viking.js

```diff
@@ -1,11 +1,204 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+
+Soldier.prototype.attack = function () {
+  return this.strength;
+}
+
+Soldier.prototype.receiveDamage = function (theDamage) {
+  this.health = this.health - theDamage;
```

Comment:
What? jaja this method is too long and is not the way to inherit from a super class but I understand you went through this with Jack